src/components/sections/ImageGallery/ImageGallery.tsx

```diff
@@ -0,0 +1,31 @@
+import { GatsbyImage } from 'gatsby-plugin-image'
+import type { IGatsbyImageData } from 'gatsby-plugin-image'
+import React from 'react'
+import { Carousel } from '@vtex/store-ui'
+import clsx from 'clsx'
+
+import * as styles from './ImageGallery.module.css'
+
+type Props = {
+  images: IGatsbyImageData[]
+  alt: string
+}
+
+export default function ImageGallery(props: Props) {
+  const images = props.images.map((image, idx) => (
+    <GatsbyImage
+      key={`${idx}-${image.images.fallback?.src}`}
+      image={image}
+      alt={props.alt}
+    />
+  ))
+
+  return (
+    <>
+      <div className={clsx(styles.container, styles.onlyDesktop)}>{images}</div>
+      <div className={clsx(styles.container, styles.onlyMobile)}>
+        <Carousel>{images}</Carousel>
```

Comment:
The pagination visual representation is going to be added in another PR?
<img width="279" alt="Screen Shot 2021-08-20 at 12 52 14" src="https://user-images.githubusercontent.com/19983991/130260075-940b0054-61fa-47ff-b301-cd1c309f71e2.png">

Reply:
in another PR, because it's the same pagination on carousel at home.